### PR TITLE
add progress bar to http_get

### PR DESCRIPTION
**Pull request by WeixiongLin**
I have discussed with the maintainer in [issue-490](https://github.com/allenai/scispacy/issues/490), and encouraged to raise this PR.

**Gist**:
To use scispacy, some of the resources are fetched from web url, e.g. [URL of tfidf_vectors_sparse](https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/data/linkers/2023-04-23/umls/tfidf_vectors_sparse.npz). And the size of the resources could be up to 500M, thereby taking several miutes to complete downloading. Somehow scispacy download quietly without signals for users, so thet can't tell whether the progress stucks or not.

So I add a progress bar to the function `http_get` to let users know where they are.

Scispacy helps me a lot in my research, thanks for your contribution to our community. And thanks for your review.

### CODE REVIEW

**Review by dakinggg — APPROVED** [review]
Thank you @WeixiongLin!

File: scispacy/file_cache.py [hunk 1]
```diff
@@ -96,9 +97,14 @@ def filename_to_url(filename: str, cache_dir: Optional[str] = None) -> Tuple[str
 
 def http_get(url: str, temp_file: IO) -> None:
     req = requests.get(url, stream=True)
+    total = int(req.headers.get("content-length", 0))
+    pbar = tqdm(total=total, unit="iB", unit_scale=True, unit_divisor=1024)
     for chunk in req.iter_content(chunk_size=1024):
         if chunk:  # filter out keep-alive new chunks
             temp_file.write(chunk)
```
Comment (dakinggg) [inline scispacy/file_cache.py:104]: I think you forgot to delete this line on this new pr. I think that is what is causing the test failures.

**Comment by WeixiongLin** [comment]
I have fix the problem, thanks for your help.

**Review by dakinggg — APPROVED** [review]
Thank you!

**Pull request merged**